stories/core/Select.stories.tsx

```diff
@@ -195,23 +195,20 @@ DisabledWithSelectedValue.args = {
   ],
 };
 
-export const ManyItems: Story<SelectProps<number>> = (args) => {
-  const { placeholder = 'Placeholder text', options, ...rest } = args;
+export const ManyItems: Story<Partial<SelectProps<number>>> = (args) => {
+  const { placeholder = 'Placeholder text', ...rest } = args;
   const [value, setValue] = useState<number | undefined>(undefined);
   return (
     <div style={{ minHeight: 350 }}>
       <Select<number>
-        options={
-          options ||
-          [...new Array(20)].map((_, index) => ({
-            label: `Item #${index}`,
-            value: index,
-          }))
-        }
         value={value}
         onChange={setValue}
         placeholder={placeholder}
         {...rest}
+        options={[...Array(20).fill(null)].map((_, index) => ({
```

Comment:
Why needed to move and set Partial props too?

Reply:
I think that might not be needed anymore. I was playing around with the array (before I switched to `Array(N).fill()` syntax).

I'll revert next time I run this branch.

Reply:
yeah..I guess what I dont like with `Partial` it kinda changes whole original interface.. Not sure it has huge influence for Storybook, though..